### ISSUE TITLE
Dnd looks unfinished

### ISSUE THREAD

**Issue by stemey** [atoha/dbootstrap#76]
In the theme previewer there is a "1" on top of the label being dragged. It seems misplaced.

**Comment by martinpengellyphillips** [comment]
Yes, haven't done any DnD specific theming yet. Thanks for the report!
